components/chip/chip_constants.js

```diff
@@ -10,6 +10,13 @@ export const CHIP_CLOSE_BUTTON_SIZE_MODIFIERS = {
   md: '',
 };
 
+export const CHIP_ICON_SIZES = {
+  xs: '200',
+  sm: '300',
+  md: '300',
+};
```

Comment:
When I choose an icon _before_ choosing a Chip size, it's not aligned to the default icon size for the default Chip size.

<img width="105" alt="image" src="https://user-images.githubusercontent.com/1165933/202301242-50bf0ce4-77af-47e5-bb8d-806648854d81.png">